torch/_C/_autograd.pyi

```diff
@@ -90,6 +90,8 @@ def _disable_profiler() -> _ProfilerResult: ...
 def _profiler_enabled() -> bool: ...
 def _add_metadata_json(key: str, value: str) -> None: ...
 def _kineto_step() -> None: ...
+def get_sequence_nr() -> int: ...
```

Comment:
very minor nit - these can probably be private as well. (you could leave these for a follow-up PR if you prefer)

Reply:
Done - I changed the name to _get_sequence_nr().  I went ahead and removed get_thread_id(), it turns out it wasn't needed afterall.